test/videojs.ima.test.js

```diff
@@ -58,3 +58,42 @@ test('ad plays on good ad tag', function() {
     start();
   }, 5000);
 });
+
+test('video continues after ad was skipped', function() {
+  var options = {
+    id: 'video',
+    adTagUrl: 'http://pubads.g.doubleclick.net/gampad/ads?sz=640x360&' +
+        'iu=/6062/iab_vast_samples/skippable&ciu_szs=300x250,728x90&impl=s&' +
+        'gdfp_req=1&env=vp&output=xml_vast2&unviewed_position_start=1&' +
+        'url=[referrer_url]&correlator=[timestamp]'
+  }
+
+  var readyForPrerollCallback = function() {
+    player.ima.addEventListener(google.ima.AdEvent.Type.SKIPPABLE_STATE_CHANGED, function() {
+      this.skip();
+    })
+    player.ima.start();
+  };
+
+  player.ima(options, readyForPrerollCallback);
+
+  var contentResumeCount = 0;
+  player.ima.onContentResumeRequested_ = function() {
+    contentResumeCount++;
+  }
+  var contentCompleteCount = 0;
+  player.ima.onAdComplete_ = function() {
+    contentCompleteCount++;
```

Comment:
This is ad complete, not content complete, so the variable name should reflect that.